about/team/team/mpolimera.md

```diff
@@ -3,8 +3,12 @@ layout: about
 title: "Dr. Mugdha Polimera"
 ---
 
-<img src="{{ site.baseurl }}/about/img/ads_logo.png" height="200" width="200" alt="Image" style="float: left; margin: 4px 10px 0px 0px; border: 1px solid #000000;">
+<img src="{{ site.baseurl }}/about/img/mpolimera.jpg" height="200" width="200" alt="Image" style="float: left; margin: 4px 10px 0px 0px; border: 1px solid #000000;">
```

Comment:
Update the `src` attribute's path: `src="{{ site.baseurl }}/about/team/img/mpolimera.jpg"`